packages/browser/src/modules/logto-client.ts

```diff
@@ -0,0 +1,28 @@
+import { OidcConfigResponse, Requester } from '@logto/js';
+
+export type LogtoConfig = {
+  endpoint: string;
+  clientId: string;
+  scopes?: string[];
+  resources?: string[];
+  usingPersistStorage: false;
```

Comment:
should be boolean?

Reply:
I just follow the SDK convention. Maybe need discussion?
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/10594507/153175899-5e6147d5-0e56-44aa-9b81-4cd73896a193.png">

Reply:
But the type should be boolean, with `false` being the default value of that boolean property.
So I think it should be `optional` as well, as only optional property can have default value. 

Thus:
```
usingPersistStorage?: boolean;
```

Reply:
Updated. PTAL @gao-sun @demonzoo